testcases/peak/ckan_post-update_1u_1s.js

```diff
@@ -0,0 +1,56 @@
+import http from 'k6/http';
+import exec from 'k6/execution';
+import {HTTP_OPTIONS} from "../../config/httpConfig.js";
+import {CONFIG} from "../../config/config.js"
+import {getUrl, wrapSetup, deleteAllRessources, randomIntBetween} from "../../util/resources.js";
+import {createPeakScenarios} from "./scenarios.js";
+
+
+export let options = CONFIG.options;
+
+const URL = getUrl();
+const CKAN_API_URL = URL + CONFIG.ckanAPIPath;
+const DATASET_NAME = CONFIG.datasetName;
+let scenarioDuration = 300;
+let scenarioPause = 30;
+let scenarioNamePrefix = `${__ENV.SYSTEM}_ckan_post-update_1u_1s`
+
+let vus = [1, 2, 4, 6, 10, 42]
```

Comment:
Warum weichen hier die vus von den vus (1, 2, 4, **_8, 16, 32, 48_**) in den anderen Tests ab?

Reply:
Die Verteilung (1,2,4,6,10,42) bei den Schreibzugriffen entstand aus einer akademischen Überlegung von Jörg Domaschka (Uni Ulm), die er uns und Ihren Kollegen Herr Weitzel, Herr Tamanini und Herr Rost in einer Mail am 04.11.2021 zukommen lassen hat. Ein Auszug hiervon: 

„In IDAP-846 wurde beschrieben, dass es bis Ende 2022 maximal 250 Sensoren und 150 Nutzer auf der Datenplattform geben wird. Mit einem Sicherheitspuffer von 20% kommt man dann auf 300 Sensoren und 180 Nutzer. Man nimmt nun gemeinhin an, dass die Sensoren sich auf die Benutzer nicht gleich verteilen, sondern das wenige Power-User den Großteil der Sensoren besitzen während die meisten Nutzer nur einen oder keinen Sensor besitzt (Zipfian Verteilung).

Unter der obigen Annahme wurden die 180 Nutzer deshalb in sieben unterschiedlich starke Gruppen unterteilt: (1,3,8,17,32,48,71 Mitglieder). Alle Mitglieder einer Gruppe besitzen in dem Modell die gleiche Anzahl an Sensoren (42, 10, 6,4,2,1,0). Folglich kommt man dann auf eine Gesamtzahl Sensoren von 1*42+3*10+...+71*0=300.

Mit den Tests für den Schreibzugriff soll nun jeweils ein Nutzer aus jeder Nutzergruppe abgebildet werden.“

Reply:
Vielen Dank für das ausführliche Feedback